packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -667,6 +667,28 @@ function processDebugValues(
   }
 }
 
+function handleRenderFunctionError(error: any): void {
+  // original error might be any type.
+  const isError = error instanceof Error;
+  if (isError && error.name === 'ReactDebugToolsUnsupportedFeatureError') {
+    throw error;
+  }
+  // If the error is not caused by an unsupported feature, it means
+  // that the error is caused by user's code in renderFunction.
+  // In this case, we should wrap the original error inside a custom error
+  // so that devtools can show a clear message for it.
+  const messgae: string =
+    isError && error.message
+      ? error.message
+      : 'Error rendering inspected component';
+  // $FlowFixMe: Flow doesn't know about 2nd argument of Error constructor
+  const wrapperError = new Error(messgae, {cause: error});
```

Comment:
Instead of using `$FlowFixMe` (we don't want to disable flow if we can) you can always do

```
const wrapperError = new Error(message);
wrapperError.cause = error;
```

Reply:
That would be a different flow error
<img width="861" alt="image" src="https://user-images.githubusercontent.com/1001890/160891461-28903584-50e9-4786-976f-f09fa94f2f87.png">

Reply:
Ooh weird let me take a look

Reply:
Yeah seems like Flow doesn't support this 😭 We could ask them to add it but in the meantime let's leave your OG way